moveit_simple_controller_manager/include/moveit_simple_controller_manager/follow_joint_trajectory_controller_handle.h

```diff
@@ -64,9 +63,14 @@ class FollowJointTrajectoryControllerHandle : public ActionBasedControllerHandle
     if (!controller_action_client_)
       return false;
 
+    /*
+     * This should never happen as the moveit_simple_controller_manager now loads
+     * MULTI_DOF_FOLLOW_JOINT_CONTROLLER_HANDLE for multi-dof joints
+     */
```

Comment:
this is a misleading comment -- this should not happen unless someone misconfigures their controllers -- which is exactly what this check is here for.